Userland/Libraries/LibJS/AST.cpp

```diff
@@ -2434,4 +2434,71 @@ void ScopeNode::add_hoisted_function(NonnullRefPtr<FunctionDeclaration> hoisted_
 {
     m_hoisted_functions.append(hoisted_function);
 }
+
+Value ImportStatement::execute(Interpreter& interpreter, GlobalObject&) const
+{
+    InterpreterNodeScope node_scope { interpreter, *this };
+    VERIFY(!"Modules are not fully supported yet!");
```

Comment:
I wanted to give a message that we do not support importing yet but maybe this should just be a debug message?